.github/workflows/llvm-project-tests.yml

```diff
@@ -50,10 +51,12 @@ jobs:
       # python3.10.6 libraries instead of the 64-bit libraries when building
       # lldb.  Using this setup-python action to make 3.10 the default
       # python fixes this.
+      # CMake choose the most recent version of Python (32bit) which installed
+      # in 'windows-latest' (VS2022) GitHub image.
```

Comment:
The newly added code and comment now conflict with the original comment above. Please merge and rewrite both comments.

Reply:
Okay, I'm updating the comment.